.github/workflows/cbmc-update.yml

```diff
@@ -51,6 +51,8 @@ jobs:
             sed -i "s/^CBMC_MINOR=.*/CBMC_MINOR=\"$CBMC_LATEST_MINOR\"/" kani-dependencies
             sed -i "s/^CBMC_VERSION=.*/CBMC_VERSION=\"$CBMC_LATEST\"/" kani-dependencies
             git diff
+            # install the newer CBMC version
+            scripts/setup/ubuntu/install_cbmc.sh
```

Comment:
Maybe add a `./` prefix to be consistent with other script invocations in this file.